templates/observations.html

```diff
@@ -40,7 +41,11 @@
     $(function () {
 
       // Init tooltips
-      $('[data-toggle="tooltip"]').tooltip();
+      {% if request.user_agent.is_mobile or request.user_agent.is_tablet %}
+          $('[data-toggle="tooltip"]').tooltip({trigger: "click"});
```

Comment:
@rkowalski4 try a much simpler solution - AFAIR Bootstrap 3 allows setting multiple triggers, and the best one is reliably chosen. `trigger: 'hover click'` should work. I'm not certain, but if it does (test please) we could avoid finicky user agent detection.

Also, check if we don't have Modernizer already in place or other feature detection working client-side with Bootstrap 3. 

Something like:

```
var is_touch_device = ("ontouchstart" in window) || window.DocumentTouch && document instanceof DocumentTouch;
```

would also be simpler than relying on UA.